core/src/main/scala/caliban/Value.scala

```diff
@@ -29,8 +29,15 @@ object ResponseValue extends ValueJsonCompat {
     override def toString: String = values.mkString("[", ",", "]")
   }
   case class ObjectValue(fields: List[(String, ResponseValue)])    extends ResponseValue {
-    override def toString: String =
+    override def toString: String            =
       fields.map { case (name, value) => s""""$name":${value.toString}""" }.mkString("{", ",", "}")
+
+    override def hashCode: Int               = fields.toSet.##
+    override def equals(other: Any): Boolean =
```

Comment:
This is needed because Play on Scala 2.13 parses the list in reverse order, which makes the fields don't line up. But since this is really a map anyway, it might make more sense to compare the values as a set (there is the edge caase of json maps being order sensitive, but not really sure how to deal with that fact and still have some notion of equality).